x/bank/commands/account.go

```diff
@@ -0,0 +1,85 @@
+package commands
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	crypto "github.com/tendermint/go-crypto"
+	rpcclient "github.com/tendermint/tendermint/rpc/client"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/examples/basecoin/app"
```

Comment:
doesn't seem like this should be in x/bank if it needs to import basecoin!

Reply:
there is a comment that we need to refactor handling tx type.
left as a todo for rigel to reflect what he wants since friday.
no answer yet